lib/Runtime/Library/EngineInterfaceObject.cpp

```diff
@@ -192,8 +192,11 @@ namespace Js
     {
         typeHandler->Convert(commonNativeInterfaces, mode, 38);
 
-        ScriptContext* scriptContext = commonNativeInterfaces->GetScriptContext();
-        JavascriptLibrary* library = scriptContext->GetLibrary();
+        JavascriptLibrary* library = commonNativeInterfaces->GetScriptContext()->GetLibrary();
+
+        library->EnsureIndexOfByteCode();
+
+        library->DefaultCreateFunction(library->indexOfByteCode->GetNestedFunctionForExecution(0), 1, commonNativeInterfaces, Js::PropertyIds::builtInJavascriptArrayEntryIndexOf);
```

Comment:
> library->indexOfByteCode->GetNestedFunctionForExecution(0)

Is it going to be a common pattern to hardcode the offset like this? After a certain number of functions I think this would be difficult to maintain.